pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +124,18 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if errors.IsNotFound(err) {
+					runningStrategy, invalid := vm.RunStrategy()
+					if invalid != nil {
+						return invalid
+					}
+					if runningStrategy == kubevirtV1.RunStrategyHalted {
+						return fmt.Errorf("Error pausing VirtualMachineInstance %s. VirtualMachine %s is not set to run", vmiName, vmiName)
+					}
+					return fmt.Errorf("Error pausing VirtualMachineInstance %s was not found", vmiName)
```

Comment:
nit `..., it was not found`

Reply:
Isn`t it redundant ? VM test was not found vs VM test, it was not found  ?

Reply:
maybe just add a comma after `VirtualMachineInstance`, `Error pausing VirtualMachineInstance test was not found` does not sound right?

Reply:
ah I see 2 info in one long sentence